examples/asr/asr_vad/audio_to_feature_rttm.py

```diff
@@ -0,0 +1,379 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Given a manifest containing audio files, first extract and save audio features, 
+then perform voice activity detection (VAD) and save the detected speech segments 
+as RTTM files. The manifest file must contain `audio_filepath` field:
+{"audio_filepath": "/path/to/audio.wav"}
+
+Example usage with default configurations:
+```bash
+python audio_to_rttm.py \
+    --config-path=./configs --config-name=vad_inference_postprocess \
+    manifest_filepath=/PATH/TO/MANIFEST.json output_dir=./vad_output
+```
+"""
```

Comment:
Please add some comments where they can set the configs. Maybe give one example on a common usage which users may need to change in the config?